Source/WebKit/UIProcess/API/Cocoa/_WKApplicationManifest.h

```diff
@@ -74,6 +74,7 @@ WK_CLASS_AVAILABLE(macos(10.13.4), ios(11.3))
 
 @property (nonatomic, readonly, nullable, copy) NSString *rawJSON;
 @property (nonatomic, readonly) _WKApplicationManifestDirection dir;
+@property (nonatomic, readonly, nullable, copy) NSString *lang;
```

Comment:
I need to understand the big picture. Why are we adding all these getters to this Objective-C class. What is this SPI for? Is this a building block for some feature we are working on?